templates/components/carousel.html

```diff
@@ -60,9 +60,11 @@
     </a>
     {{/if}}
     {{/each}}
-    {{#and arrows (if carousel.slides.length '>' 1)}}
+    {{#if carousel.slides.length '>' 1}}
+    {{#if arrows}}
     <button aria-label="{{lang 'carousel.arrowAriaLabel'}} {{carousel.slides.length}}" class="js-hero-next-arrow slick-next slick-arrow"></button>
-    {{/and}}
+    {{> components/carousel-content-announcement slides_length=carousel.slides.length}}
```

Comment:
Do we really need announcement only with arrows?

Reply:
moved component above arrow conditions